### ISSUE TITLE
TensorFlow version?

### ISSUE THREAD

**Issue by ntelo007** [yiskw713/boundary_loss_for_remote_sensing#3]
Hi,
thanks for your work. Could you please help me write down a TensorFlow or Keras version of this loss function?

**Comment by yiskw713** [comment]
Hi, ntelo007

I don't know how to use a TensorFlow and Keras, but I can write this loss with numpy.
If this helps you, I'll try it later.

**Comment by ntelo007** [comment]
Thank you very much, that would be great!

**Comment by rishavroy1264bitmesra** [comment]
@ntelo007 Have done Tensorflow implementation for the same, works awesome for my segmentation training : https://github.com/rishavroy1264bitmesra/Tensorflow-Boundary-Loss-for-Remote-Sensing-Imagery-Semantic-Segmentation

Regards
